src/content/docs/kubernetes-pixie/kubernetes-integration/advanced-configuration/monitor-services/monitor-services-running-kubernetes.mdx

```diff
@@ -0,0 +1,342 @@
+
+We support the following services running on Kubernetes:
+
+* [Apache](/docs/integrations/host-integrations/host-integrations-list/apache-monitoring-integration) (does not report [inventory data](/docs/integrations/new-relic-integrations/getting-started/understand-use-data-infrastructure-integrations#overview))
+* [Cassandra](/docs/integrations/host-integrations/host-integrations-list/cassandra-monitoring-integration)
+* [Couchbase](/docs/integrations/host-integrations/host-integrations-list/couchbase-monitoring-integration)
+* [Elasticsearch](/docs/integrations/host-integrations/host-integrations-list/elasticsearch-monitoring-integration)
+* [HAProxy](/docs/integrations/host-integrations/host-integrations-list/haproxy-monitoring-integration)
+* [HashiCorp Consul](/docs/integrations/host-integrations/host-integrations-list/hashicorp-consul-monitoring-integration)
+* [JMX](/docs/integrations/host-integrations/host-integrations-list/jmx-monitoring-integration)
+* [Kafka](/docs/integrations/host-integrations/host-integrations-list/kafka-monitoring-integration)
+* [Memcached](/docs/integrations/host-integrations/host-integrations-list/memcached-monitoring-integration)
+* [MongoDB](/docs/integrations/host-integrations/host-integrations-list/mongodb-monitoring-integration)
+* [MySQL](/docs/integrations/host-integrations/host-integrations-list/mysql-monitoring-integration)
+* [NGINX](/docs/integrations/host-integrations/host-integrations-list/nginx-monitoring-integration)
+* [PostgreSQL](/docs/integrations/host-integrations/host-integrations-list/postgresql-monitoring-integration)
+* [RabbitMQ](/docs/integrations/host-integrations/host-integrations-list/rabbitmq-monitoring-integration) (does not report [inventory data](/docs/integrations/new-relic-integrations/getting-started/understand-use-data-infrastructure-integrations#overview))
+* [Redis](/docs/integrations/host-integrations/host-integrations-list/redis-monitoring-integration)
+* [SNMP](/docs/integrations/host-integrations/host-integrations-list/snmp-monitoring-integration)
+
+## Enable monitoring of services using the Helm Chart [#enable]
+
+Monitoring services in Kubernetes works by leveraging our standard On-Host integrations and an autodiscovery mechanism to point them to pods with a specified selector.
+
+To enable our Kubernetes integration to monitor one or more services:
+
+### Get the base configuration [#integration-config]
+
+You can get an example configuration file for our integrations that can run in K8s from the following links:
+
+<CollapserGroup>
+  <Collapser
+  id="integration-configs"
+  title="Get the config YAML for the integration"
+  >
+    For the services you want to monitor, follow the links to GitHub to get the YAML snippets you'll need for the next step:
+
+    * [Apache](https://github.com/newrelic/nri-apache/blob/master/apache-config.yml.k8s_sample)
```

Comment:
have we checked that the format of the linked files is not too outdated?

We could do a quick pass on them

Reply:
We definitely should, as most of them are outdated. But I think that if we block the PR because of that we won't release until next month :(